drivers/network/meson/ethernet.h

```diff
@@ -141,16 +141,32 @@
 #define DESC_TXSTS_OWNBYDMA         (1 << 31)           /* Descriptor is owned by the DMA of the GMAC Subsystem. */
 
 /* Tx control bit definitions */
-#define DESC_TXCTRL_TXINT		    (1 << 31)           /* Sets Transmit Interrupt after the present frame has been transmitted. */
-#define DESC_TXCTRL_TXLAST		    (1 << 30)           /* Buffer contains the last segment of the frame. */
-#define DESC_TXCTRL_TXFIRST		    (1 << 29)           /* Buffer contains the first segment of a frame. */
-#define DESC_TXCTRL_TXCRCDIS		(1 << 26)           /* GMAC does not append the Cyclic Redundancy Check (CRC) to the end of the transmitted frame.*/
-#define DESC_TXCTRL_TXRINGEND		(1 << 25)           /* Descriptor list reached its final descriptor. DMA must loop around. */
-#define DESC_TXCTRL_TXCHAIN		    (1 << 24)           /* Second address in the descriptor is the Next Descriptor address rather than the second buffer address. */
-#define DESC_TXCTRL_SIZE2MASK		(0x3ff800)
-#define DESC_TXCTRL_SIZE2SHFT		(11)
-#define DESC_TXCTRL_SIZE1MASK		(0x7FF)
-#define DESC_TXCTRL_SIZE1SHFT		(0)
+#define DESC_TXCTRL_TXINT           (1 << 31)           /* Sets Transmit Interrupt after the present frame has been transmitted. */
+#define DESC_TXCTRL_TXLAST          (1 << 30)           /* Buffer contains the last segment of the frame. */
+#define DESC_TXCTRL_TXFIRST         (1 << 29)           /* Buffer contains the first segment of a frame. */
+#define DESC_TXCTRL_TXCRCDIS        (1 << 26)           /* GMAC does not append the Cyclic Redundancy Check (CRC) to the end of the transmitted frame.*/
+#define DESC_TXCTRL_TXRINGEND       (1 << 25)           /* Descriptor list reached its final descriptor. DMA must loop around. */
+#define DESC_TXCTRL_TXCHAIN         (1 << 24)           /* Second address in the descriptor is the Next Descriptor address rather than the second buffer address. */
+#define DESC_TXCTRL_SIZE2MASK       (0x3ff800)
+#define DESC_TXCTRL_SIZE2SHFT       (11)
+#define DESC_TXCTRL_SIZE1MASK       (0x7FF)
+#define DESC_TXCTRL_SIZE1SHFT       (0)
+
+/* GMAC FLOW CTRL defines */
```

Comment:
These `GMAC_FLOW_CTRL` definitions repeat some earlier in the file.  We should decide which to keep, and adjust the places they're used accordingly.

Reply:
I think I'll move them up with the other definitions. Originally I designed the file so that the register definitions came first, followed by the descriptor definitions.